### PR TITLE
Resolve VCSWP-18414 (Number management changes for java sdk)

### PR DESCRIPTION
**Pull request by amahadaya**
Number management changes for java sdk

### CODE REVIEW

**Review by smedavarapu-vail — CHANGES_REQUESTED** [review]
I mentioned few spellings and empty functions.

**Review by edgeston — CHANGES_REQUESTED** [review]
I think the overall PR looks good. Had some questions regarding potential typos, formatting, tests, and missing descriptions

**Review by edgeston — CHANGES_REQUESTED** [review]
Removing approval

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 None
 
+<a name="5.4.0"></a>
+
+## [5.4.0] 2023-06-06
+
+### Changed
```
Comment (ngarg-vail) [inline CHANGELOG.md:16]: Added

**Review by ngarg-vail — CHANGES_REQUESTED** [review]

**Review by ngarg-vail — APPROVED** [review]

**Pull request merged**